x/operator/client/cli/query.go

```diff
@@ -456,3 +460,106 @@ func GetOptInfo() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+func QuerySnapshotHelper() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QuerySnapshotHelper <avsAddr>",
+		Short: "Get the voting power snapshot helper for the avs",
+		Long:  "Get the voting power snapshot helper for the avs",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !common.IsHexAddress(args[0]) {
+				return xerrors.Errorf("invalid avs address,err:%s", types.ErrInvalidAddr)
+			}
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QuerySnapshotHelperRequest{
+				Avs: strings.ToLower(args[0]),
+			}
+			res, err := queryClient.QuerySnapshotHelper(context.Background(), req)
+			if err != nil {
+				return err
+			}
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
+
+func QueryAllSnapshot() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryAllSnapshot <avsAddr>",
+		Short: "Get the all voting power snapshots for the avs",
+		Long:  "Get the all voting power snapshots for the avs",
```

Comment:
```suggestion
		Long:  "Get the latest 6 epochs voting power snapshots for the avs",
```

Reply:
Seems like it should be the `unbondingDuration+1`